<launch>
  <group ns="pcc">
	<node pkg="shipcon" name="main_motor" type="motor">
		<param name="IpAddr" value="192.168.39.10" />
		<param name="Port" value="50001" />
		<param name="SubnameControlVal" value="/pcc/ztest_controller/motor_command" />
	</node>

	<node pkg="shipcon" name="bow_thruster" type="bowthruster" output="screen">
		<param name="IpAddr" value="192.168.39.13" />
		<param name="Port" value="50003" />
		<param name="SubnameControlVal" value="/Autonode/Subname/BowThruster" />
	</node>

	<node pkg="shipcon" name="rudder" type="single_rudder">
		<param name="IpAddr" value="192.168.39.11" />
		<param name="Port" value="50002" />
		<param name="SubnameControlVal" value="/pcc/ztest_controller/rudder_command" />		
	</node>

	<node pkg="shipcon" name="radio_control" type="radio_control">
		<param name="IpAddr" value="192.168.39.100" />
		<param name="Port" value="50000" />
	</node>

	<node pkg="shipcon" name="ztest_controller" type="ztest_controller">
		<param name="SubnameGyro" value="/pcc/gyro/gyro" />
		<param name="SubnameRadioControl" value="/pcc/radio_control/mode_value" />
	</node>

	<node pkg="shipcon" name="gyro" type="gyro_jae_jg35fd" output="screen">
		<param name="serial/port_name" value="/dev/ttyUSB0" />
	</node>

	<node pkg="shipcon" name="gps" type="gps_pub.py" />
	
  </group>
</launch>
